header/entropy/entropy.hpp

```diff
@@ -0,0 +1,46 @@
+/************************/
+/**     Libraries      **/
```

Comment:
change "Libraries" to "Includes". Libraries are a different thing

Reply:
changes brought